tests/integration/test_fim/test_files/test_report_changes/test_large_changes.py

```diff
@@ -147,38 +116,35 @@ def test_large_changes(filename, folder, original_size, modified_size, get_confi
                  the test will verify that the generated FIM event contains in its 'content_changes' field the proper
                  value depending on the test case.
 
-    wazuh_min_version: 4.2.0
+    wazuh_min_version: 4.5.0
 
     tier: 1
 
     parameters:
-        - filename:
-            type: str
-            brief: Name of the testing file to be created.
-        - folder:
-            type: str
-            brief: Path to the directory where the testing files will be created.
-        - original_size:
-            type: int
-            brief: Size of the testing file in bytes before being modified.
-        - modified_size:
-            type: int
-            brief: Size of the testing file in bytes after being modified.
-        - get_configuration:
+        - configuration:
+            type: dict
+            brief: Configuration values for ossec.conf.
+        - metadata:
+            type: dict
+            brief: Test case data.
+        - test_folders:
+            type: dict
+            brief: List of folders to be created for monitoring.
+        - set_wazuh_configuration:
             type: fixture
-            brief: Get configurations from the module.
-        - configure_environment:
+            brief: Set ossec.conf configuration.
+        - create_monitored_folders_module:
             type: fixture
-            brief: Configure a custom environment for testing.
-        - configure_local_internal_options_module:
+            brief: Create a given list of folders when the module starts. Delete the folders at the end of the module.
+        - configure_local_internal_options_function:
             type: fixture
-            brief: Configure the local internal options file.
-        - restart_syscheckd:
+            brief: Set local_internal_options.conf file.
+        - restart_syscheck_function:
             type: fixture
-            brief: Clear the 'ossec.log' file and start a new monitor.
-        - wait_for_fim_start:
+            brief: restart syscheckd daemon, and truncate the ossec.log.
```

Comment:
Please, update the description without the specific name file.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4219/commits/d664f239c801fb219b9c529f5320e36c9508961b